src/xdocs/config_whitespace.xml

```diff
@@ -843,6 +847,9 @@ package com.puppycrawl.
 
 import com.puppycrawl.tools.
     checkstyle.api.AbstractCheck;
+
+import static java.math.
```

Comment:
please add comment " //violation" on line where you expect it. for other cases too in this example.